api/v1/model/src/commonMain/kotlin/PagedResponse.kt

```diff
@@ -50,6 +50,21 @@ data class PagingOptions(
     val sortProperties: List<SortProperty>? = null
 )
 
+@Serializable
+data class PagingData(
+    /** The limit for the number of items to return. */
+    val limit: Int,
```

Comment:
Why is this just an `Int` if the below two are `Long`s?

Reply:
Because requesting more than max int entities would not be feasible, but the database could contain more entries.